app/sql-commands.sql

```diff
@@ -0,0 +1,167 @@
+-- Table: public.issues
+
+-- DROP TABLE public.issues;
+
+CREATE TABLE public.issues
+(
+  id integer NOT NULL DEFAULT nextval('"issues_ID_seq"'::regclass),
+  url character varying(100),
+  repository_url character varying(100),
+  git_issue_id integer,
+  title character varying(100),
+  user_id integer,
+  body text,
+  CONSTRAINT issues_pkey PRIMARY KEY (id)
+)
+WITH (
+  OIDS=FALSE
+);
+ALTER TABLE public.issues
+  OWNER TO postgres;
+
+
+
+-- Table: public.permissions
+
+-- DROP TABLE public.permissions;
```

Comment:
Zrobilem sobie skrypty zeby zreprodukowac baze danych w domu. Przenioslem te informacje to pliku readme.md i usunalem ten plik.